README.rst

```diff
@@ -40,6 +40,32 @@ Remember, if co-installing multiple CWL implementations then you need to
 maintain which implementation ``cwl-runner`` points to via a symbolic file
 system link or `another facility <https://wiki.debian.org/DebianAlternatives>`_.
 
+Running tests locally
+---------------------
+
+-  Running unnittests ``(/tests)``:
+
+.. code:: bash
+
+    pip install tox
+    tox # to run all tests
```

Comment:
Developers might not have all the versions of Python installed that we test using `tox`, let's give them instructions on how to run the tests w/o tox